Resources/Prototypes/Entities/Objects/Consumable/Food/Containers/condiments.yml

```diff
@@ -357,6 +358,7 @@
     - type: SolutionContainerVisualizer
       maxFillLevels: 6
       fillBaseName: bottle-alpha-
+  - type: TrashOnEmpty
```

Comment:
same here, just on the base

Reply:
the condiment ones already are the base objects, there's just a few different base condiment bottles